schema/partner_show.js

```diff
@@ -113,6 +114,24 @@ const PartnerShowType = new GraphQLObjectType({
         return gravity(path, options);
       },
     },
+    meta_image: {
+      type: Image.type,
+      resolve: ({ id, partner, image_versions, image_url }) => {
+        if (image_versions && image_versions.length && image_url) {
+          return Image.resolve({ image_versions, image_url });
+        }
+
+        return gravity(`partner/${partner.id}/show/${id}/artworks`, {
+          published: true,
+        })
+          .then(artworks => {
+            const shareable = find(artworks, { can_share_image: true });
+            if (shareable && shareable.images) {
+              return Image.resolve(getDefault(shareable.images));
+            }
+          });
```

Comment:
This didn't occur to me earlier but I'm pretty sure this can just be:

```javascript
Image.resolve(getDefault(find(artworks, { can_share_image: true })));
```